app/src/Navigators/Main.tsx

```diff
@@ -1,63 +1,123 @@
 /**
  * @file Defines the main screens that have tabs at the bottom of the app, e.g. Projects, Events.
  */
-
 import React from 'react'
-import { createBottomTabNavigator } from '@react-navigation/bottom-tabs'
+import {
+  createBottomTabNavigator,
+  BottomTabNavigationOptions,
+} from '@react-navigation/bottom-tabs'
+import { useRoute } from '@react-navigation/native'
 import { ListContainer } from '@/Containers'
-import Entypo from 'react-native-vector-icons/Entypo'
 import { ListType } from '@/Containers/ListContainer'
-import { useTheme } from '@/Hooks'
-import ProfileContainer from '@/NativeBase/Containers/ProfileContainer'
-import VerticalStackContainer from '@/NativeBase/Containers/VerticalStackContainer'
+import {
+  ProfileContainer,
+  VerticalStackContainer,
+} from '@/NativeBase/Containers'
+
+import SelectionIcons from '@/NativeBase/Assets/Icons/SelectionIcons'
+
+import { View, useColorMode } from 'native-base'
+import StaTheme from '@/NativeBase/Theme/StaTheme'
 
 const Tab = createBottomTabNavigator()
 
-// @refresh reset
+interface CustomNavBarIconProps extends BottomTabNavigationOptions {
+  focused?: boolean
+}
+
+const NavBarIcon = ({ focused }: CustomNavBarIconProps) => {
+  const { colorMode } = useColorMode()
+  const route = useRoute()
+  let iconName = route.name
+
+  switch (route.name) {
+    case 'Home':
+      iconName = 'home'
+      break
+    case 'Projects':
+      iconName = 'six-dots'
+      break
+    case 'Events':
+      iconName = 'ticket'
+      break
+    case 'Profile':
+      iconName = 'user'
+      break
+  }
+
+  return (
+    <View
```

Comment:
On my Android emulator the text below the icons looks very close to the bottom of the screen -- might need some extra bottom padding in order to make this more comfortable and match the Figma designs?

![Screenshot 2023-02-20 18 29 51 - Display 2](https://user-images.githubusercontent.com/25086222/220178767-44686e42-05e0-417d-a2f3-18b358aa87ec.png)

Reply:
Thanks Joe, you're right. These were the default position of the tab labels. The inactive colour is also a lighter default. Looking better at our design it is indeed different, have added custom label styling now and removed the vague grey line at the top of the nav bar.